dts/arm/nxp/nxp_imx8m_m4.dtsi

```diff
@@ -180,7 +210,7 @@
 		};
 	};
 };
-
+https://github.com/zephyrproject-rtos/zephyr/pull/77736
```

Comment:
what is it for?

Reply:
Whoops, this shouldn't be there. Interesting, that all checks passed